src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
+      {todos.map(({ title, completed }) => (
```

Comment:
Need to add a key prop to the mapped div element to avoid warning and improve rendering performance.